modules/webgl/src/classes/program.js

```diff
@@ -276,21 +282,9 @@ export default class Program extends Resource {
   // Binds textures
   // Note: This is currently done before every draw call
   _bindTextures() {
-    for (const uniformName in this.uniforms) {
-      const uniformSetter = this._uniformSetters[uniformName];
-
-      if (uniformSetter && uniformSetter.textureIndex !== undefined) {
-        let uniform = this.uniforms[uniformName];
-
-        if (uniform instanceof Framebuffer) {
-          uniform = uniform.texture;
-        }
-        if (uniform instanceof Texture) {
-          const texture = uniform;
-          // Bind texture to index
-          texture.bind(uniformSetter.textureIndex);
-        }
-      }
+    for (const uniformName in this._texturesUniforms) {
+      const textureIndex = this._uniformSetters[uniformName].textureIndex;
+      this.uniforms[uniformName].bind(textureIndex);
```

Comment:
Doesn't bind return a new function? That is just thrown away here, so this is a no-op?

Reply:
This code wasn't changed in this PR.